### ISSUE TITLE
[WADTex] WadTex 1.2 does not produce expected output

### ISSUE THREAD

**Issue by akorov** [MTrop/DoomTools#86]
Hi,
Using WadTex v 1.2 to dump textures from a wad/resource pack produces a .txt file with only the 2-row comment header.
I'm using standalone version (doomtools-cmd) on Windows 10.
Reverting to release containing version 1.1 (doomtools-cmd-2022.04.21.071908288) fixes the issue.

**Example execution with both versions - dumping OTEX, you can see massively different size in txt files:**

```
PS F:\Doooooom\wads\dev\merge> cd .\DoomTools\
PS F:\Doooooom\wads\dev\merge\DoomTools> .\wadtex.cmd --version
WADTex v1.2.0 by Matt Tropiano (using DoomStruct v2.14.2)
PS F:\Doooooom\wads\dev\merge\DoomTools> .\wadtex.cmd ..\OTEX_1.1.WAD --export ..\TEXTURE2.txt -v
Wrote `..\TEXTURE2.txt`.
PS F:\Doooooom\wads\dev\merge\DoomTools> ls ..\TEXTURE2.txt


    Directory: F:\Doooooom\wads\dev\merge


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        2022-11-19     14:19             93 TEXTURE2.txt


PS F:\Doooooom\wads\dev\merge\DoomTools> cd ..\doomtools-cmd-2022.04.21.071908288\
PS F:\Doooooom\wads\dev\merge\doomtools-cmd-2022.04.21.071908288> .\wadtex.cmd --version
WADTex v1.1.0 by Matt Tropiano (using DoomStruct v2.14.2)
PS F:\Doooooom\wads\dev\merge\doomtools-cmd-2022.04.21.071908288> .\wadtex.cmd ..\OTEX_1.1.WAD --export ..\TEXTURE2.txt -v
Wrote `..\TEXTURE2.txt`.
PS F:\Doooooom\wads\dev\merge\doomtools-cmd-2022.04.21.071908288> ls ..\TEXTURE2.txt


    Directory: F:\Doooooom\wads\dev\merge


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----        2022-11-19     14:20         189371 TEXTURE2.txt


PS F:\Doooooom\wads\dev\merge\doomtools-cmd-2022.04.21.071908288>
```

![image](https://user-images.githubusercontent.com/11968110/202848471-98b531b9-fdb2-45e2-adc5-d7fc9710248d.png)

**Comment by MTrop** [comment]
Interesting. I get the full output if "Open from WAD File" in the GUI editor. I wonder what the difference is? Hmm....

Thanks for alerting me! I'll get to look into it later this week.

**Comment by MTrop** [comment]
Looks like it was broken since July-ish. Fixed in commit 53172179ab7375a69b89564a0e9df46bbe81ac5b.
